charts/umbrella/values.yaml

```diff
@@ -681,6 +681,148 @@ managed-identity-wallet:
       nginx.ingress.kubernetes.io/cors-allow-methods: GET
       nginx.ingress.kubernetes.io/enable-cors: "true"
 
+bpdm:
+  enabled: true
```

Comment:
by default we add all the chart with enabled set to false

for testing we have 2 different values files, could you please add and enable bpdm in the one for the shared services? https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/b6d50476ab4689b64e4b6418b538f9669407b9eb/charts/values-test-shared-services.yaml